content/tracing/faq.md

```diff
@@ -19,6 +19,10 @@ customnav: tracingnav
 
 Tracing data is stored for about 36 hours.
 
+### How frequent does the trace-agent send stats?
```

Comment:
hmmm, sounds strange to me. maybe "How often does the trace agent send stats?"

Reply:
👍